students/mariusz_michalczyk/lesson_07_strings/num_words.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == '__main__':
+    input_str = input()
+    print(input_str.count(' ') + 1)
```

Comment:
It works on all the test cases and it's an obvious way to use count, so I guess that's what was meant.

But I would argue that you could have multiple spaces between, if it was production code.